## Patches for microb-engine
## Oleg Romashin <oleg.romashin@nokia.com>
##
## DP: Fix building with new make.
## [Bug 325148] perl syntax error from rules.mk: make 3.81 changes backslash-newline behavior inside single-quoted strings

Index: mozilla/security/coreconf/rules.mk
===================================================================
--- mozilla.orig/security/coreconf/rules.mk
+++ mozilla/security/coreconf/rules.mk
@@ -926,17 +926,17 @@
 		    unlink(($$tmpname));                                      \
 		}                                                             \
 	    } elsif ("$<" ne "$(DEPENDENCIES)") {                             \
 		print "$(MAKE): *** No rule to make target $<.  Stop.\n";     \
 		exit(1);                                                      \
 	    }
 
 .DEFAULT:
-	@$(PERL) -e '$(PERL_DEPENDENCIES_PROGRAM)'
+	@`$(PERL) -e '$(PERL_DEPENDENCIES_PROGRAM)'`
 endif
 
 #############################################################################
 # X dependency system
 #############################################################################
 
 ifdef MKDEPENDENCIES
 
